source/slang/slang.vcxproj

```diff
@@ -225,7 +225,6 @@
     <ClInclude Include="syntax-defs.h" />
     <ClInclude Include="syntax-visitors.h" />
     <ClInclude Include="syntax.h" />
-    <ClInclude Include="tmpC85.tmp.h" />
```

Comment:
What in the world?

Reply:
I'm not sure how that happened, but it was part of a PR I did #885 . I'm assuming I ran premake and that file was around and was included. It is odd in so far as I never remember creating such a file, so perhaps a tool generated it.